pkg/resources/installTypesHelper.go

```diff
@@ -0,0 +1,21 @@
+package resources
+
+import (
+	integreatlyv1alpha1 "github.com/integr8ly/integreatly-operator/apis/v1alpha1"
+)
+
+func IsRHOAM(installType integreatlyv1alpha1.InstallationType) bool {
+	return installType == "managed-api" || installType == "multitenant-managed-api"
+}
+
+func IsRHMI(installType integreatlyv1alpha1.InstallationType) bool {
+	return installType == "managed"
+}
+
+func IsRHOAMMultitenant(installType integreatlyv1alpha1.InstallationType) bool {
```

Comment:
Not seeing this being used, Guess its because we don't have any multi tenant config in the operator yet

Reply:
Yes, it will be used by other functionalities provided